src/content/about/oasis-plus-comparison.mdx

```diff
@@ -11,7 +11,7 @@ import TableHeaderBorderless from "@components/TableHeaderBorderless.astro"
 import TableCellBorderless from "@components/TableCellBorderless.astro"
 export const components = {ul: IconList, li: CheckListItem,  table: TableBorderless, th: TableHeaderBorderless, td: TableCellBorderless};
 
-The OASIS+ family of contracts builds on the success of the OASIS and OASIS Small Business or SB contracts, offering federal agencies access to highly qualified contractors for their comprehensive and complex non-IT services solutions worldwide.
+The OASIS+ family of contracts builds on the success of the OASIS and OASIS Small Business contracts, offering federal agencies access to highly qualified contractors for their comprehensive and complex non-IT services solutions worldwide.
```

Comment:
Should the "OASIS SB" acronym be introduced here since it's used later on in the text?

Either as:

```suggestion
The OASIS+ family of contracts builds on the success of the OASIS and OASIS Small Business (OASIS SB) contracts, offering federal agencies access to highly qualified contractors for their comprehensive and complex non-IT services solutions worldwide.
```

or

```suggestion
The OASIS+ family of contracts builds on the success of the OASIS and OASIS Small Business (SB) contracts, offering federal agencies access to highly qualified contractors for their comprehensive and complex non-IT services solutions worldwide.
```